eng/sdl-tsa-vars.config

```diff
@@ -0,0 +1,12 @@
+-SourceToolsList @("policheck","credscan")
+-TsaInstanceURL https://devdiv.visualstudio.com/
+-TsaProjectName DEVDIV
+-TsaNotificationEmail aspnetcore-build@microsoft.com
+-TsaCodebaseAdmin REDMOND\kevinpi
+-TsaBugAreaPath "DevDiv\ASP.NET Core"
+-TsaIterationPath DevDiv
+-TsaRepositoryName AspNetCore
+-TsaCodebaseName AspNetCore
+-TsaOnboard $True
+-TsaPublish $True
+-PoliCheckAdditionalRunConfigParams @("UserExclusionPath < $(Build.SourcesDirectory)/eng/PoliCheckExclusions.xml")
```

Comment:
@jcagme is this the right syntax?

Reply:
If it worked before it should work since I'm passing the contents of the file wholesale as in https://github.com/dotnet/aspnetcore/pull/21153/files/60482346e29075a668155f0afaf6fe1ad6c4d1ca#diff-097cdf55e1a931b74fbfe48d7e41c8beL721